static/app/components/replays/replayContext.tsx

```diff
@@ -431,6 +431,10 @@ export function Provider({
         events: events ?? [],
         // common to both
         root,
+        context: {
+          sdkName: replay?.getReplay().sdk.name,
+          sdkVersion: replay?.getReplay().sdk.version,
```

Comment:
`getReplay` this is going throgh some list to find the integration right?
No easy way in TS to (pseudo-C#):

```
if replay?.getReplay is replayIntegration {
  sdkName: replayIntegration.sdk.name,
  sdkVersion: replayIntegration.sdk.version,
```
and avoid the double lookup?

Reply:
`getReplay()` just returns `this._replayRecord` -- no integration lookup!

Reply:
https://github.com/getsentry/sentry/blob/b965109ad3fd7da0af9e824a7338b22213dc0263/static/app/utils/replays/replayReader.tsx#L518-L520